### PR TITLE
feat(pie-icons): DSW-2192 add and update icons

### PR DESCRIPTION
**Pull request by leksaBoiko**
### Benefits
- offer-filled
- offer-filled-large

### Currency
- krone
- krone-large

### Functionality
- archive
- archive-large

### Pay
- account-credit-filled
- account-credit-filled-large

### People
- user-cancelled
- user-cancelled-large
- user-move
- user-move-large

### Restaurant
- restaurant-cancelled
- restaurant-cancelled-large

### Travel & Transport
- ebike
- ebike-large
- ebike-filled
- ebike-filled-large

### Pay
- account-credit
- account-credit-large

## Author Checklist (complete before requesting a review)
- [x] I have performed a self-review of my code
- [x] I have added thorough tests where applicable (unit / component / visual)
- [x] I have reviewed the `PIE Docs` PR preview

### Reviewer 1 - @raoufswe 
- [x] I have reviewed the `PIE Docs` PR preview
- [x] If there are visual test updates, I have reviewed them

### Reviewer 2 - @xander-marjoram 
- [x] I have reviewed the `PIE Docs` PR preview
- [x] If there are visual test updates, I have reviewed them

### CODE REVIEW

File: .changeset/quiet-dragons-deliver.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+"pie-docs": minor
+---
+
+[Updated] - Icon categories to follow alphabetical order.
```
Comment (leksaBoiko) [inline .changeset/quiet-dragons-deliver.md:5]:
```suggestion
[Updated] - Icons to follow alphabetical order.
```

File: packages/tools/pie-icons/src/assets/social/twitter-circle-filled-large.svg
Comment (xander-marjoram) [inline packages/tools/pie-icons/src/assets/social/twitter-circle-filled-large.svg:1]: Just checking, is the main purpose of this change to make the X transparent?

File: packages/tools/pie-icons/src/assets/payment/tenbis.svg
Comment (xander-marjoram) [inline packages/tools/pie-icons/src/assets/payment/tenbis.svg:1]: Is anyone using this icon currently? Renaming is technically a breaking change. I can't remember if we discussed just duplicating the icon with a different name

File: .changeset/thirty-squids-sparkle.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+- ebike
+- ebike-large
+- ebike-filled
+- ebike-filled-large
+
+[Updated] - Icons
+
+## Pay
+- account-credit
+- account-credit-large
```
Comment (xander-marjoram) [inline .changeset/thirty-squids-sparkle.md:46]: Are these changes missing?

Comment (leksaBoiko) [reply .changeset/thirty-squids-sparkle.md:46]: Good catch! Wrong pics name in changeset file, should be `account-credit-filled`

File: .changeset/thirty-squids-sparkle.md [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+## Travel & Transport
+- ebike
+- ebike-large
+- ebike-filled
+- ebike-filled-large
+
+[Updated] - Icons
+
+## Pay
+- account-credit
```
Comment (leksaBoiko) [inline .changeset/thirty-squids-sparkle.md:45]:
```suggestion
- account-credit-filled
```

File: .changeset/thirty-squids-sparkle.md (hunk 2 above)
Comment (leksaBoiko) [inline .changeset/thirty-squids-sparkle.md:46]:
```suggestion
- account-credit-filled-large
```

File: packages/tools/pie-icons/src/assets/social/twitter-circle-filled-large.svg
Comment (leksaBoiko) [reply packages/tools/pie-icons/src/assets/social/twitter-circle-filled-large.svg:1]: No, the prev icon was broken:
Screenshot 2024-08-12 at 12 44 12

File: packages/tools/pie-icons/src/assets/payment/tenbis.svg
Comment (leksaBoiko) [reply packages/tools/pie-icons/src/assets/payment/tenbis.svg:1]: Good catch!

**Review by raoufswe — APPROVED** [review]

**Review by xander-marjoram — APPROVED** [review]

**Pull request merged**